.eslintrc

```diff
@@ -1,31 +1,18 @@
 {
-  "extends": [
-    "okonet",
-    "prettier"
-  ],
-  "plugins": [
-    "prettier"
-  ],
+  "extends": ["okonet/node"],
+  "parserOptions": {
+    "sourceType": "module"
```

Comment:
The only reason is the rollup config. @TrySound is it worth using imports?

Reply:
But source code is also esm. What's the problem?